src/ucs/async/signal.c

```diff
@@ -101,10 +101,26 @@ ucs_async_signal_set_fd_owner(pid_t dest_tid, int fd)
 #endif
 }
 
+static int
+ucs_async_signal_sys_timer_create_portable(clockid_t clk, struct ksigevent *restrict evp, timer_t *restrict res)
```

Comment:
line too long

Reply:
You know, IBM stopped making punch cards in the 80s... (;-).

OK, it's your code, so it's your style rules.